forwards_paper.tex

```diff
@@ -274,13 +270,36 @@ \subsection*{Simulation benchmarks}
 execution and fitness calculations were parallelized.}
 \end{figure}
 
+We also performed a more limited set of simulations without natural selection.  The total run times are shown in
+Figure~\ref{sfig:rawspeed_nosel} and show the same qualitative behavior as simulations with selection
+(Figure~\ref{fig:runtimes_selection}).  The relative improvement due to pedigree tracking is again substantial in these
+simulations (Figure~\ref{sfig:speedup_nosel}).  In fact, we see more of a benefit to pedigree tracking here with \fwdpy{}
+than we did with selection (Figure~\ref{fig:relative_speedup_selection}).  The reason is that the fitness function exits
+close to instantly in simulations without selection that are based on \fwdpp{}, meaning that \fwdpy{} is doing little
+more than generating random numbers and book-keeping in Figures~\ref{sfig:rawspeed_nosel}
+and~\ref{sfig:speedup_nosel}.
+
+
 \begin{figure}
     \includegraphics[]{sims/speedup}
     \caption{\label{fig:relative_speedup_selection}Relative performance improvement due to pedigree tracking. This figure
     shows the ratio of the run time tracking neutral mutations over the run time tracking the pedigree. Data points are taken
 from Figure~\ref{fig:runtimes_selection} for simulations that ran to completion using both methods.}
 \end{figure}
 
+The peak RAM use in gigabytes (GB) is shown for the simulations with selection in Figure~\ref{sfig:ramsel} aand for the
+simulations without selection in Figure~\ref{sfig:ramnosel}.  The general picture is that pedigree tracking requires
+more RAM.  With \fwdpy{}/\fwdpp{}, the reason for this behavior is clear.  When pedigree tracking, a non-recombinant
```

Comment:
Hm - but RAM usage when tracking pedigrees depends mostly on how often we run simplify, no?  So, is this even a useful point to make?

Reply:
Or, rather: this seems like a useful point to make, but perhaps "pedigree tracking requires more RAM" should say instead "As we implemented these simulations, pedigree tracking required more RAM."  ... ?

Reply:
> Yes, that is true, but I think it will be true fairly generally. Simplifying every generation will reduce the speedups by a fair bit, based on my experiments.

Totally; just as written it sounds like pedigree tracking *always* uses more RAM, which clearly depends on both how many neutral mutations there are and how often simplification happens.